src/Modules/OrchardCore.Commerce/Views/TieredPricePart.cshtml

```diff
@@ -0,0 +1,15 @@
+@model TieredPricePartViewModel
+
+@if (Model.Tiers?.Any() == true)
+{
+    <div class="product-price-tiers">
+        <div>@T["Price:"] @Model.DefaultPrice.ToString()</div>
+        <div>@T["Tiers:"]</div>
+        <ul>
+            @foreach (var tier in Model.Tiers)
+            {
+                <li>@tier.Quantity - @tier.UnitPrice</li>
```

Comment:
This seems a bit obscure, I'd prefer to use a text like "Starting from {0} units".

Also `PriceTier.UnitPrice` is just a `decimal`, so we lose the expected currency formatting like how many decimal places should be displayed. Rather convert it into `Amount` so its `ToString()` overload can be used.